docs/running_docker.rst

```diff
@@ -84,15 +84,17 @@ GITHUB_TOKEN:
     https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line
 
 CONNECTED_APP_CLIENT_ID:
-    This represents the client id of the connected app that MetaDeploy will use for authenticating to any persistent org.
+    This represents the consumer key of the connected app that MetaDeploy will use for authenticating to any persistent org.
```

Comment:
@davisagli This language ("consumer key") seems to match what I see in the Connected App settings -- is there a reason to use "client id" instead?

Reply:
@jgerigmeyer Sure, it makes sense to change it here to match what is shown in the connected app UI. But generally they are used interchangeably, and all our other apps use client id for the name of the config var & setting, so I wouldn't want to change it there.